src/org/jitsi/impl/neomedia/rtcp/NewRTCPPacketParser.java

```diff
@@ -0,0 +1,66 @@
+package org.jitsi.impl.neomedia.rtcp;
+
+import org.jitsi.service.neomedia.*;
+import org.jitsi.util.*;
+
+import java.io.*;
+import java.util.*;
+
+/**
+ * Created by bbaldino on 8/23/17.
+ */
+public class NewRTCPPacketParser
+{
+    /**
+     * The {@link Logger} used by the {@link NewRTCPPacketParser} class and its
+     * instances for logging output.
+     */
+    private static final Logger logger
+        = Logger.getLogger(NewRTCPPacketParser.class);
+
+    public static NewRTCPPacket[] parse(RawPacket packet)
+    {
+        // PR-NOTE(brian): should we do some check/verification that this is, in fact,
+        // a compound rtcp packet?  Or can we expect to only be called with
+        // compound packets?
```

Comment:
Unless I'm missing something, non-compound packets can be treated as a special case of compound packets (with only one packet)

Reply:
but the iterator will only work on compound packets, is that right?